### PR TITLE
Rh/salesforce refresh token

### PR DESCRIPTION
**Pull request by Gambarou**
### Briefly describe your changes:

Added the server-to-server JWT bearer flow for Salesforce so we no longer have to manually update the access token.
Added an admin route that displays a temp admin dashboard that needs to be worked on. The chart displays data from salesforce but it's not perfect and needs to be reworked.
Added a popup that displays an error message if a client tries to pay an invoice that doesn't exist.
There are other changes.

**Jira issue number and link:**

**Type of change:**
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Please answer the following:

**1. What is the current behavior?**
*If the feature did not exist, please say so.*
We had to manually update the access token in sales force
**2. What is the new behavior?**
The access token is updated automatically
**3. What tests did you run?**

### Checklist before requesting a review
- [x] I have performed a self-review of my code.
- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests for my code.
- [x] I have resolved all merge conflicts.

### CODE REVIEW

File: bill-bot-baggins/app/(main)/(routes)/invoice/page.tsx
Comment (cchrty) [inline bill-bot-baggins/app/(main)/(routes)/invoice/page.tsx:1]: tmeporary

File: bill-bot-baggins/app/api/salesforce-GraphQL/route.ts
Comment (cchrty) [inline bill-bot-baggins/app/api/salesforce-GraphQL/route.ts:1]: rename file for auth/token route

Comment (cchrty) [reply bill-bot-baggins/app/api/salesforce-GraphQL/route.ts:1]: ...and invoice data

**Review by cchrty — APPROVED** [review]
great job on the admin dashboard!

**Review by lhodges3 — APPROVED** [review]
looks great Robert!

**Pull request merged**